core/tests/puppeteer-acceptance-tests/utilities/common/puppeteer-utils.ts

```diff
@@ -410,6 +412,46 @@ export class BaseUser {
   getCurrentUrlWithoutParameters(): string {
     return this.page.url().split('?')[0];
   }
+
+  /**
+   * This function checks the exploration accessibility with help of explorationID.
+   */
+  async expectExplorationToBeAccessibleByUrl(
+    explorationId: string | null
+  ): Promise<void> {
+    if (!explorationId) {
+      throw new Error('ExplorationId is null');
+    }
+    const explorationUrlAfterPublished = `${baseURL}/create/${explorationId}#/gui/Introduction`;
+    try {
+      await this.page.goto(explorationUrlAfterPublished);
+      showMessage('Exploration is accessible with the URL, i.e. published.');
+    } catch (error) {
+      throw new Error('The exploration is not public.');
+    }
+  }
+
+  async expectExplorationToBeNotAccessibleByUrl(
```

Comment:
Missing comments for this function

Reply:
done

Reply:
This comment is the same as the one above. Make sure the comment describes exactly what the function does.

Reply:
There is no need to add comments for each method. If the method name is clear enough to explain its purpose or functionality, comments can be skipped.
/cc @seanlip , @imchristie

Reply:
That reasoning applies to line comments. But in general each function should have JSDoc (and in this case it seems reasonable to have that because "to be not accessible by URL" is not clear).

In general if a reviewer asks for function-level comments, that's probably a signal that it's not clear and worth providing.

Reply:
Done